cmd/config/env_var_reader.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/flyteorg/flyte/flyteidl/clients/go/admin"
+	"github.com/flyteorg/flyte/flytestdlib/config"
+)
+
+const flyteAdminEndpoint = "FLYTE_ADMIN_ENDPOINT"
+
+type FuncType func() error
+
+var funcMap = map[string]FuncType{flyteAdminEndpoint: updateAdminEndpoint}
```

Comment:
Could we have a name which represent the meaning instead of funcMap?

Reply:
How do you think about envToUpdater